### DISCUSSION TITLE
APU MASTER SW On/Off via Simconnect

### DISCUSSION THREAD

**Discussion by GRnice** [flybywiresim/aircraft#3324] [Help] [answered]
Hi,

I would like to set to ON/OFF the Apu Master SW & APU start buttons via Simconnect but I don't found any functional command ('APU_STARTER',...).
 Any idea ?

used A3nx version: 0.5.2

Simconnect helper used: https://github.com/odwdinc/Python-SimConnect

Thank you :)

**Comment by hotshotp** [comment]
@davidwalschots has changed that LocalVar with the newest apu changes. Maybe he knows

**Comment by davidwalschots** [comment, marked as answer]
Indeed! As @hotshotp suggested there have been changes related to this. You'll have to use the following local vars:

APU MASTER SW: `L:A32NX_APU_MASTER_SW_ACTIVATED` (bool)
APU START: `L:A32NX_APU_START_ACTIVATED` (bool)

**Reply by GRnice** [reply]
Ok thank you 😀 !

Last question, to update these vars I must use the simconnect.dll of MSFS SDK ?

**Reply by davidwalschots** [reply]
I don't know. I'm using an abstraction made by @devsnek to communicate with MSFS. Maybe they can help you.
